Detectors/TOF/base/src/Geo.cxx

```diff
@@ -19,3 +20,478 @@ constexpr Float_t Geo::HEIGHTS[NPLATES][NMAXNSTRIP];
 constexpr Float_t Geo::DISTANCES[NPLATES][NMAXNSTRIP];
 constexpr Bool_t Geo::FEAWITHMASKS[18];
 constexpr Float_t Geo::ROOF2PARAMETERS[3];
+
+void Geo::GetDetID(Float_t* pos, Int_t* det)
+{
```

Comment:
Hi @noferini , sorry, I did not understand what is the problem, the proposal was just to use instead of ````Int_t *det``` array a struct with 5 data members. Anyway, this was not for the efficiency but just for the safety and self-documentation of the code.